backend/apis/parse-docket-pdf.utils.js

```diff
@@ -60,7 +60,7 @@ function parseCharges(lines, sections) {
       });
     } else if (line.startsWith("Disposition:")) {
       const numeric = /\d/;
-      lastNumericIndex = 0;
+      let lastNumericIndex = 0;
```

Comment:
👍 good catch

Reply:
For future reference, I knew how to fix it because I got this error:
`ReferenceError: lastNumericIndex is not defined`